### PR TITLE
[136] Readme for Workflow example and integrations

### PR DESCRIPTION
**Pull request by CamilleLetavernier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,35 @@
...
+
+```
+yarn install
+```
+
+And the GLSP Server (Which includes the Workflow Diagram Server example). See [`glsp-server`](https://github.com/eclipse-glsp/glsp-server#building) for instructions.
+
+Once both the Server and the Client packages are built, you can start both:
+
+- Server: in `examples/org.eclipse.glsp.example.workflow`, start `org.eclipse.glsp.example.workflow.launch.WorkflowServerLauncher.java` with these arguments: `--websocket --port=8081` (Or use the included `Start_Workflow_Example_Server_(Websocket).launch` Launch Configuration from Eclipse).
```
Comment (planger) [inline README.md:25]: Maybe it is worth mentioning that the server can also be simply started from command line after building as documented here: https://github.com/eclipse-glsp/glsp#building-and-running-the-glsp-backend
This way, interested users wouldn't have to set up an IDE?

File: README.md [hunk 2]
```diff
@@ -2,6 +2,35 @@
...
+- Client: Open the `examples/workflow-standalone/app/diagram.html` file in your favorite browser
+
+### Where to find the sources?
+
+In addition to this repository, the related source code can be found here:
```
Comment (planger) [inline README.md:30]:
```suggestion
In addition to this repository, the source code of the example diagram server can be found here: https://github.com/eclipse-glsp/glsp-server/tree/master/examples/org.eclipse.glsp.example.workflow
```

File: README.md [hunk 3]
```diff
@@ -2,6 +2,35 @@
...
+
+-   https://github.com/eclipse-glsp/glsp-server
```
Comment (planger) [inline README.md:32]: I'd prefer just mentioning the link in the paragraph above instead of having a single-bullet bullet list.

**Review by planger — COMMENTED** [review]
Thanks, Camille! Looks good to me!
Just two suggestions below. What do you think?

File: README.md (hunk 1 above)
Comment (CamilleLetavernier) [reply README.md:25]: There are too many readmes spread all around :) I wasn't aware of this part of the doc

File: README.md (hunk 3 above)
Comment (CamilleLetavernier) [reply README.md:32]: I followed the structure for all other READMEs (The glsp-client is the only one with a single related repo); but removing the list is also fine

**Review by planger — APPROVED** [review]

**Pull request merged**